.circleci/tests.sh

```diff
@@ -92,4 +92,30 @@ done
 
 echo ""
 echo "Found trace \"$TRACE_ID\"."
+
+echo "Checking for broken links"
+
+curl http://${TRACES_SERVER_HOST}:2020/ -s -f -o /dev/null   || { echo "link is not available" ; exit 1; }
```

Comment:
1. as there is a pattern here for deep link `services/service/<id>/ [overview | endpoints | traces | metrics]`, do you want to consider doing by a simple function and list as an argument.
e.g 
```
links-to-test = ["services", "backend"]
func test-deep-links(links-to-tests) {
for each:links-to-tests:
 test base link
 test suffix links //services/service/123456-123456/overview
}
```
2. If there is any change in endpoints, we will have to change this script too. cc: @skjindal93

Reply:
I'd rather follow @kotharironak, it would also allow to tell what link is not available. That feedback is necesary.

Reply:
okay.